From 6d193776421db948e4201a60cf239fdae62c196d Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.i.king@gmail.com>
Date: Thu, 4 Jan 2024 09:52:57 +0000
Subject: [PATCH] libtypec: utils: add missing return at end of function

Function typecstatus_power_contract is missing a return 0 at the
end of the function, currently it is returning nothing and so
it may be returning a garbage value.

Signed-off-by: Colin Ian King <colin.i.king@gmail.com>
Upstream-Status: Backport [upstream commit 6d19377]
---
 utils/typecstatus.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/utils/typecstatus.c b/utils/typecstatus.c
index 5e16e74..bfb4142 100644
--- a/utils/typecstatus.c
+++ b/utils/typecstatus.c
@@ -208,7 +208,7 @@ int typecstatus_power_contract()
                     printf("\tNo Power Contract on port %d\n",i);        
             }
         }
-
+	return 0;
 }
 
 /* Check all typec ports */
-- 
2.43.2

